config/crd/bases/instancemgr.keikoproj.io_instancegroups.yaml

```diff
@@ -62,373 +64,24 @@ spec:
             submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
           type: string
         metadata:
-          properties:
```

Comment:
Not sure why all the changes to this file happend, including the above ones.
Maybe you need to run `make generate` & `make manifests` again?
The only changes to this file should be the addition of type `managedPolicies`

Reply:
Looks like we are using different controller-gen versions and I was trying to figure out the version but controller-get doesn't have "version" command to display it. Anyway I deleted my controller-gen from GOPATH/bin since you are installing as part of make generate if it is not there and now its all good but probably that might be an issue if developer using different version in future as it keeps deleting other sections